spec/workers/podcast_episodes/create_worker_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe PodcastEpisodes::CreateWorker, type: :worker do
+  include_examples "#enqueues_on_correct_queue", "high_priority", 1
```

Comment:
What's the `1` in this example?

Should this be:
```ruby
include_examples "#enqueues_on_correct_queue", "high_priority", [781, { "foo" => "bar" }]
```
to simulate the args?

Reply:
actually yea it should, kind of surprised the spec didn't fail. I misunderstood that to be the size of the queue which it isn't now that I re-look at the [definition](https://github.com/thepracticaldev/dev.to/blob/master/spec/workers/shared_examples/enqueues_on_correct_queue.rb)

I will dig in and fix it up

Reply:
oh right I see why it works because enqueuing will work but processing the job will fail (but that's not what this shared example tests)